daffodil-core/src/main/scala/org/apache/daffodil/dpath/Expression.scala

```diff
@@ -1687,6 +1688,16 @@ case class FunctionCallExpression(functionQNameString: String, expressions: List
 
       //End typeValueCalc related functions
 
+      case (RefQName(_, "DoubleFromRawLong", DFDLX), args) => {
```

Comment:
That name of the function should still start with a lowercase letter. It's only the class name that should have a capital D.